File: tasks/implementations/node/tasks/csv_parsing.js
const fs = require('fs');
module.exports = function(size, fixturesRoot) {
  const lines = fs.readFileSync(`${fixturesRoot}/generated/csv/${size}.csv`, 'utf8').trim().split(/\n/);
  let total = 0;
  for (let i = 1; i < lines.length; i++) {
    const parts = lines[i].split(',');
    total += Number(parts[1]) + Number(parts[2]);
  }
  console.log(total);
};

File: tasks/implementations/rust/tasks/csv_parsing.rs
use std::fs;

pub fn run(size: &str, fixtures_root: &str) {
    let text = fs::read_to_string(format!("{}/generated/csv/{}.csv", fixtures_root, size)).unwrap();
    let mut total = 0i64;
    for line in text.lines().skip(1) {
        let parts: Vec<&str> = line.split(',').collect();
        total += parts[1].parse::<i64>().unwrap() + parts[2].parse::<i64>().unwrap();
    }
    println!("{}", total);
}
